src/js/index.js

```diff
@@ -0,0 +1,114 @@
+import "../css/style.css";
+import {shapes} from "./entities/shapes";
+import {Point} from "./entities/points";
+let mousePosition;
+let offsets;
+let currentShape;
+let canvas;
+let context;
+
+window.onload = () => {
+    offsets = [];
```

Comment:
you don't need `offets` just use
you can create `const boundingClientRect = canvas.getBoundingClientRect();` and use it to find a current position on canvas